class-tgm-plugin-activation.php

```diff
@@ -439,7 +439,7 @@ public function install_plugins_page() {
 				<?php $plugin_table->prepare_items(); ?>
 
 				<?php
-				if ( isset( $this->message ) ) {
+				if ( isset( $this->message ) && is_string( $this->message ) && '' !== $this->message ) {
```

Comment:
#wceu public is a bit too diverse for my hard-core PHP defensive coding talk (_The big "Why equal doesn't equal" Quiz_), but maybe we should do with a select group in a hallway track or something....